.env.example

```diff
@@ -26,6 +26,8 @@ MAIL_PORT=2525
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
 MAIL_ENCRYPTION=null
+MAIL_FROM_ADDRESS=regis@monicahq.com
```

Comment:
Please replace this by `ValidEmailAddress` or something, so this email is never used by people ever.